content/tools/data.js

```diff
@@ -216,13 +216,39 @@ const tools = [
   },
   {
     title: 'Extended Drought',
-    desc: `Projections of temperature, precipitation and a set of VIC variables from two 30 year 
-    drought periods.`,
+    desc: `Explore projections of temperature, precipitation and a set of hydrological variables 
+      from two 20 year drought periods.`,
     categories: ['Temperature', 'Precipitation'],
```

Comment:
add category `Snowpack`

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/125/commits/4a775d5f11f50beae431b50793e45c76fb1d40d5